src/edgeDebugSession.ts

```diff
@@ -2,14 +2,14 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
-import {ChromeDebugSession} from 'vscode-chrome-debug-core';
+import {ChromeDebugSession, IChromeDebugSessionOpts} from 'vscode-chrome-debug-core';
 import {EdgeDebugAdapter} from './edgeDebugAdapter';
 
 export class EdgeDebugSession extends ChromeDebugSession {
 
     public constructor(targetLinesStartAt1: boolean, isServer: boolean = false) {
```

Comment:
You can probably simplify things by getting rid of this class and using `ChromeDebugSession.getSession` with some args from edgeDebug.ts. See how vscode-chrome-debug does it for an example.